make/ide/idea/jdk/idea.gmk

```diff
@@ -46,13 +46,14 @@ else #with SPEC
   endif
 
   idea:
-	$(ECHO) "SUPPORT=$(SUPPORT_OUTPUTDIR)" >> $(OUT)
-	$(ECHO) "MODULE_ROOTS=\"$(foreach mod, $(SEL_MODULES), $(call FindModuleSrcDirs,$(mod)))\"" >> $(OUT)
+	$(ECHO) "SUPPORT=$(SUPPORT_OUTPUTDIR)" > $(OUT)
+	$(ECHO) "MODULES=\"$(foreach mod, $(SEL_MODULES), module='$(mod)' moduleSrcDirs='$(call FindModuleSrcDirs,$(mod))' moduleDependencies='$(call FindTransitiveDepsForModule,$(mod))' #)\"" >> $(OUT)
```

Comment:
If you can find a way to break this line, it would be appreciated. We try to keep line length "reasonable" within the build files. (Reasonable meaning, not strict 80, but with future 3-way merges on a normal screen in mind)